scootapi/scoot.thrift

```diff
@@ -31,8 +35,9 @@ struct JobDefinition {
   2: optional JobType jobType,
 }
 
+// TODO(dbentley): rename this to... Exec? CloudExec? CloudScoot?
 service Proc {
-  Job RunJob(1: JobDefinition job) throws (
+   JobId RunJob(1: JobDefinition job) throws (
```

Comment:
why are we not returning the whole job anymore? I thought the idea was that we'd send back the Job with assigned JobIds and TaskIds.